extras/uwsgi/ideascube.conf

```diff
@@ -1,5 +0,0 @@
-[Service]
-User=ideascube
-Group=ideascube
-PAMName=ideascube
-Environment=SHELL=/bin/bash
```

Comment:
@fheslouin In another comment you mention that you were sharing that unit with another webapp.

That means your other webapp was running as the `ideascube` user, because all of uWSGI was running as that user.

But with the removal of the file above, now uWSGI will run normally as root (only the ideascube app runs as the `ideascube` user).

So if you want your other webapp to run as a specific non-root user, you'll need to adapt its configuration.

See the `ideascube.ini` configuration file, it includes the `uid` and `gid` parameters necessary for that.

But please don't add the above removed file back into Ansiblecube, that never was necessary, and we shouldn't have done it.